nixos/modules/services/mail/postfix.nix

```diff
@@ -508,6 +510,14 @@ in
         '';
       };
 
+      useDane = mkOption {
```

Comment:
Could this not be `smtp_tls_security_level` exposed directly?

Reply:
It looked like the intent of #90115 was to derive `smtp_tls_security_level` from the value of `tlsTrustedAuthorities` and/or `sslCert`, so I stuck with that.

Though I did just notice that since I developed this, #94270 has landed, which also alters the value of `smtp_tls_security_level` under different conditions ... so, cc @typetetris @petabyteboy

Reply:
The default value could be computed based on `tlsTrustedAuthorities` and `sslCert`.